src/Plugin/Field/FieldFormatter/AuthorsReferenceFormatter.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\Plugin\Field\FieldFormatter\EntityReferenceFormatterBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Plugin implementation of the Author reference fields formatter.
+ *
+ * @FieldFormatter(
+ *   id = "authors_reference_formatter",
+ *   label = @Translation("Authors reference formatter"),
+ *   field_types = {
+ *     "entity_reference_revisions"
+ *   }
+ * )
+ */
+class AuthorsReferenceFormatter extends EntityReferenceFormatterBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function defaultSettings() {
+    return [
+      'label_only' => '',
+      'rel' => '',
+      'target' => '',
+    ] + parent::defaultSettings();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function settingsForm(array $form, FormStateInterface $form_state) {
+    $elements = parent::settingsForm($form, $form_state);
+
+    $elements['label_only'] = [
+      '#type' => 'checkbox',
+      '#title' => t('Label only'),
```

Comment:
```suggestion
      '#title' => $this->t('Label only'),
```
StringTranslationTrait is a part of this class. I see that core's EntityReferenceEntityFormatter uses t(), but other formatters use $this->t(). Can we also use $this->t() here?